lib/page.js

```diff
@@ -19,19 +19,21 @@ module.exports = class Page {
 			return Promise.resolve(this);
 		}
 
-		let { headers, body } = await colonParse(this.sourcePath);
+		let { colonParse } = await metacolon;
```

Comment:
... but apparently Node v18 doesn't do top-level await yet? I thought it should, but GitHub Actions disagreed. 🤷